easygui/boxes/egstore.py

```
@@ -104,55 +95,42 @@ def restore(self):
         Where possible, those attributes will have values recovered
         from the pickled object.
         """
-        if not os.path.exists(self.filename):
-            return self
-        if not os.path.isfile(self.filename):
-            return self
-
-        try:
-            with open(self.filename, "rb") as f:
-                unpickledObject = pickle.load(f)
-
-            for key in list(self.__dict__.keys()):
-                default = self.__dict__[key]
-                self.__dict__[key] = unpickledObject.__dict__.get(key, default)
-        except:
-            pass
+        with open(self._filename, 'rb') as f:
+            store = pickle.load(f)
```

Don't ignore errors when explicitly calling `restore`.  Especially don't ignore them with a bare `except`!